lib/browser/init.ts

```diff
@@ -49,10 +49,12 @@ process.on('uncaughtException', function (error) {
   }
 
   // Show error in GUI.
-  const dialog = require('electron').dialog
-  const stack = error.stack ? error.stack : `${error.name}: ${error.message}`
-  const message = 'Uncaught Exception:\n' + stack
-  dialog.showErrorBox('A JavaScript error occurred in the main process', message)
+  import('electron')
```

Comment:
why not a static import?

Reply:
You can't `import 'electron'` half way down a file, static imports have to be at the top of a file

Reply:
is there any particular reason not to just import this at the top of the file then?

Reply:
We can't 😢 The `electron` alias is configured / setup half-way down this file in `reset-search-paths`

Reply:
ah, okay. Let's comment here about that then!